source/extensions/common/wasm/null/null_plugin.h

```diff
@@ -47,6 +47,7 @@ namespace Plugin {
 struct NullPluginRegistry {
   uint32_t (*proxy_validate_configuration_)(uint32_t root_context_id,
                                             uint32_t plugin_configuration_size) = nullptr;
+  uint32_t (*proxy_on_context_create_)(uint32_t context_id, uint32_t parent_context_id) = nullptr;
   uint32_t (*proxy_on_start_)(uint32_t root_context_id, uint32_t vm_configuration_size) = nullptr;
```

Comment:
Per the document, this should be renamed to `proxy_on_vm_start()`.

Reply:
Done.